src/routes/choice-exercises/components/SingleChoiceListItem.tsx

```diff
@@ -7,7 +7,7 @@ import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 
 const StyledText = styled.Text`
   font-family: SourceSansPro-Regular;
-  font-size: ${wp('4.3%')};
+  font-size: ${wp('4.3%')}px;
```

Comment:
see discussion here: https://github.com/styled-components/css-to-react-native/issues/40